### PR TITLE
update kind node images

### PR DESCRIPTION
**Pull request by ChristianGeie**
add comment to point out used kind version out of kind-action

### CODE REVIEW

**Comment by ChristianGeie** [comment]
@jekkel @tomrk-esteam8 need review pls.

File: .github/workflows/build_and_test.yaml [hunk 1]
```diff
@@ -43,27 +45,27 @@ jobs:
           - maj_min: v1.22
             digest: sha256:f5b2e5698c6c9d6d0adc419c0deae21a425c07d81bbf3b6a6834042f25d4fba2
```
Comment (jekkel) [inline .github/workflows/build_and_test.yaml:46]: kind 0.22.0 doesn't seem to officially support anything below 1.23 anymore. Shouldn't we delete these then?

**Review by jekkel — APPROVED** [review]

File: .github/workflows/build_and_test.yaml (hunk 1 above)
Comment (ChristianGeie) [reply .github/workflows/build_and_test.yaml:46]: Thx for pointing out. I agree to remove v1.21 and v1.22

**Comment by ChristianGeie** [comment]
I will also adjust tests as required checks in the repo config

**Review by tomrk-esteam8 — APPROVED** [review]
LGTM

**Pull request merged**